marklogic-data-hub-central/build.gradle

```diff
@@ -78,10 +78,8 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp:4.7.2"
     implementation "com.squareup.okhttp3:logging-interceptor:4.7.2"
 
-    // Excluding okhttp3 dependency as it bumps down the dependency from 4.4.0 to 3.x
-    implementation ('com.microsoft.azure:azure-keyvault-secrets-spring-boot-starter:2.2.4') {
-        exclude group: "com.squareup.okhttp3", module: "okhttp"
-    }
```

Comment:
Replaced this with azure's identity and keyvault secret java libraries as the earlier library was injecting the value from the key directly using `@Value`. 
This is not required in the case where both hubPendo and hubRetrieveCertificate is set to false.